specs/merge/fork.md

```diff
@@ -0,0 +1,106 @@
+# Ethereum 2.0 The Merge
+
+**Notice**: This document is a work-in-progress for researchers and implementers.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+- [Fork to Merge](#fork-to-merge)
+  - [Fork trigger](#fork-trigger)
+  - [Upgrading the state](#upgrading-the-state)
+  - [Initializing transition store](#initializing-transition-store)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+This document describes the process of the Merge upgrade.
+
+## Configuration
+
+Warning: this configuration is not definitive.
+
+| Name | Value |
+| - | - |
+| `MERGE_FORK_VERSION` | `Version('0x02000000')` |
+| `MERGE_FORK_EPOCH` | `Epoch(18446744073709551615)` **TBD** |
+
+## Fork to Merge
+
+### Fork trigger
+
+TBD. Social consensus, along with state conditions such as epoch boundary, finality, deposits, active validator count, etc. may be part of the decision process to trigger the fork. For now we assume the condition will be triggered at epoch `MERGE_FORK_EPOCH`.
+
+Note that for the pure Merge networks, we don't apply `upgrade_to_merge` since it starts with Merge version logic.
```

Comment:
For this to be true, we need to define an `initialize_beacon_state` function. In prior forks this function is defined as `initialize_beacon_state_from_eth1` and reads in deposits. When starting from a merged state, this function will likely need a list of initial deposits or validators along with an initial execution-layer state root.

We can deal with this in a later PR but should probably note here that a standard way to do this is currently missing from specs

Reply:
Right, we also need to do this for Altair as well. I'd make this change in a separate PR along with tests refactor and new transition process tests.

Reply:
Altair *does* have this function [redefined](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/altair/beacon-chain.md#initialize-state-for-pure-altair-testnets-and-test-vectors)